pluma/pluma-view.c

```diff
@@ -2034,13 +2032,17 @@ static gboolean
 pluma_view_button_release_event (GtkWidget *widget, GdkEventButton *event)
 {
 	if (event->button == 2)
-		middledown = FALSE;
-	else if (event->button == 3)
-		rightdown = FALSE;
+		middle_or_right_down = FALSE;
```

Comment:
Is is OK to remove everything for event->button == 3 here ?

Reply:
yes, never works the button 3 here (middle click), I did instead the function pluma_view_populate_popup

you can test that, it fixes a little issue:

1- select text

2- right button (appears the menu)

3- middle button (dissapears the menu)

4- middle button again

- the expected, with this PR: paste the text

- without this PR: nothing happens, it needs one more middle click

Reply:
Ok, got it

Reply:
Confirmed the issue and the fix.